### ISSUE TITLE
dmv.viewer.VolumeImageViewer not rendering canvas to the DOM (DICOM microscopy viewer not being displayed) 

### ISSUE THREAD

**Issue by selvesandev** [herrmannlab/slim#132]
Hello,

We are trying to integrate `dcom-microscopy-viewer` into our web portal. He've followed the slim example to do so. we have been able to performa a `searchForStudies` and extract the response to display the DICOM information in a list. 

However when we are trying to render the DICOM file in the detail (viewer) screen. The page is just blank and nothing is displayed and also there are no error messages.

**After doing some investigations I found that the `dmv.viewer.VolumeImageViewer` on `SlideViewer.js` file spits out a html elements with a canvas and renders it the DOM element.**

<img width="1550" alt="image" src="https://user-images.githubusercontent.com/21096850/201575603-8e562cb5-0130-426c-86d9-c017604e543d.png">

**However in our implementation the spitted out html does not contains `canvas` inside the html elements.** 

<img width="1101" alt="image" src="https://user-images.githubusercontent.com/21096850/201576160-c5219670-146a-430a-97ab-61ee906dc613.png">

I guess this might be the reason why the viewer is not showing up.

Few things that are different in the slim example repository and our react portal are

1. Our portal is not using typescript
2. We are using `react-router-dom@5.2.0`
3. And we are using `react-app-rewired` instead or craco because craco is not supported in older webpack version.

I've created a sample example in this git repository. [https://github.com/selvesandev/slim-integration](https://github.com/selvesandev/slim-integration)

In this Example I've trimmed out all the codes (eg: codes for drawing ROI, Sidebar views etc) and tried to make it as slim as possible.

Please help us out to track the main cause for the viewer screen not being rendered.

Thank you

**Comment by hackermd** [comment]
@selvesandev thanks for your interest in the viewer library and for reporting the issue. Were you able to fix the problem?

**Comment by selvesandev** [comment]
Hi @hackermd Yes I got the issue resolved. But the solution was quite weird :D

we are not using the `antd` library in our portal and hence I eliminated the <Layout> tags that was wrapping the <ParametrizedCaseViewer>. This was the reason why the canvas was not being rendered. 

And after I got the html elements back the canvas got rendered itself. Not sure what is the catch here 😄 

<img width="796" alt="image" src="https://user-images.githubusercontent.com/21096850/202908336-36ddaa2a-65fb-45c9-ab2a-d0768fe928b3.png">
